interface/forms/misc_billing_options/new.php

```diff
@@ -2,13 +2,12 @@
 /*
  * This program creates the misc_billing_form
  *
- * Copyright (C) 2007 Bo Huynh
- * Copyright (C) 2016 Terry Hill <terry@lillysystems.com>
- * Copyright (C) 2017 Brady Miller <brady.g.miller@gmail.com>
- *
  * @package OpenEMR
  * @author Terry Hill <terry@lilysystems.com>
  * @author Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (C) 2007 Bo Huynh
+ * @copyright Copyright (C) 2016 Terry Hill <terry@lillysystems.com>
+ * @copyright Copyright (C) 2017 Brady Miller <brady.g.miller@gmail.com>
```

Comment:
Don't forget to add yourself to these headers. Pretty much any modification allows you to add yourself as author/copyright.